safehttp/mux.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package safehttp
+
+import (
+	"errors"
+	"net/http"
+)
+
+// TODO: add the missing methods
+const (
+	// HTTP GET request
+	MethodGet = "GET"
```

Comment:
Not an initialism or an acronym, but feels it should have the same treatment as them: https://github.com/golang/go/wiki/CodeReviewComments#initialisms. `MethodGET`.

Reply:
`net/http` uses `MethodGet` https://pkg.go.dev/net/http?tab=doc#pkg-constants

Reply:
I guess these were defined before the code review comments. Makes sense for them to stay then.